b2/console_tool.py

```diff
@@ -1790,6 +1805,43 @@ def format_ls_entry(self, file_version: FileVersion, replication: bool):
         return template % tuple(parameters)
 
 
+@B2.register_subcommand
+class Rm(Ls):
+    """
+    Removes a group of files. Use with caution.
+
+    To list files to be deleted, use ``--dry``. You can also list files
+    via ``ls`` command – the listing behaviour is exactly the same.
+
+    {LSCOMMAND}
+
+    The ``--dry`` option prints all the files that will be removed.
+
+    Requires capability:
+
+    - **listFiles**
+    - **deleteFiles**
+    """
+
+    @classmethod
+    def _setup_parser(cls, parser):
+        parser.add_argument('--dry', action='store_true')
+        super()._setup_parser(parser)
+
+    def run(self, args):
+        if args.dry:
+            # Print all the files that are to be fetched.
+            super().run(args)
+            return 0
```

Comment:
```suggestion
            return super().run(args)
```
if we ever decide that ls can output some weird code (say, "123" when there is an indication that some files mighrt have been hidden from view of this client due to insufficient permissions, so the output might not be complete), then we'll want to keep this behavior for dryRun too